libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2793,6 +2793,43 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
       },
     ],
   },
+  xdcnetwork: {
+    type: "CryptoCurrency",
+    id: "XDC",
+    coinType: 550,
+    name: "XDCNetwork",
+    managerAppName: "XDCNetwork",
+    ticker: "XDC",
+    scheme: "xdc",
+    color: "#2b5876",
+    family: "ethereum",
+    ethereumLikeInfo: {
+      baseChain: "mainnet",
+      chainId: 50,
+      networkId: 50,
+      rpc: "https://rpc.xdcrpc.com",
+      chainId: 50,
```

Comment:
Duplicate `chainId` key here

Reply:
Thank you , We have updated changes in it.